data/mm/world/overworld.yml

```diff
@@ -372,7 +412,7 @@
     "Cucco Shack": "true"
     "Doggy Racetrack": "true"
   locations:
-    "Romani Ranch Epona Song": "can_use_keg && has(BOW)"
+    "Romani Ranch Epona Song": "can_use_keg"
```

Comment:
You need the bow to do the minigame I think? Or can you just wait 2 minutes and still get the check?